tasks/managed/prepare-fbc-release/prepare-fbc-release.yaml

```diff
@@ -34,6 +34,9 @@ spec:
         set -euxo pipefail
 
         SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
+        # need a copy as the snapshot file is readonly
```

Comment:
Why is it readonly? That seems odd. For example, we do a similar thing with the data file where we modify it in populate-release-notes. So if it's possible there, why would it be a problem here?

Reply:
I will crosscheck. I had also the permission denied error on the temp.json file used (line 139) which I changed to `mktemp`. It might be the case that I thought the error was in both places, but it just there. I will let you know. The bad news is that I need e2e to crosscheck if this is an actual problem or not.

Reply:
it was actually only the `temp.json` the problem. I removed the .updated part. Thanks for the heads up.